<launch>
    <group ns='nuc'>
        <!-- <node pkg='robotic_sas_auv_ros' type='node_guidance.py' name='node_guidance' output='screen'/> -->
        <node pkg='robotic_sas_auv_ros' type='node_gui.py' name='node_gui' output='screen'/>
        <!-- <node pkg='robotic_sas_auv_ros' type='node_navigation.py' name='node_navigation' output='screen'/>
        <node pkg='robotic_sas_auv_ros' type='node_control.py' name='node_control' output='screen'/> -->
    </group>
</launch>